iroh-net/src/magic_endpoint.rs

```diff
@@ -391,16 +408,15 @@ impl MagicEndpoint {
     /// If no UDP addresses are added, and the given `derp_region` cannot be dialed, it will error.
     pub async fn add_known_addrs(
         &self,
-        node_id: PublicKey,
+        peer: PublicKey,
```

Comment:
weren't we calling these `node_id`s? why the change back?

Reply:
I asked about this in #1487 
We are using peer_id through the gossip and sync code so at this point it feels more consistent than node to me. As long as we pick one I'm happy to oblige

Reply:
I see, I don't mind either way, as long as we are somewhat consistent about it 😅